Lib/zipfile.py

```diff
@@ -2131,7 +2133,7 @@ def addToZip(zf, path, zippath):
             return count
         if len(files):
             count = 0
-            with ZipFile(zip_name, mode) as zf:
+            with ZipFile(zip_name, mode, allowZip64=args.zip64) as zf:
```

Comment:
This works without a call to set_defaults?

Reply:
Yes @merwok - Because the argument parser is set to store_false on the flag ards.zip64 is automatically set to True unless the flag is seen.